terraform-tests/terraform_tests_suite_test.go

```diff
@@ -17,15 +17,15 @@ func TestTerraformTests(t *testing.T) {
 }
 
 var (
+	workingDir string
+
 	awsSecretAccessKey = os.Getenv("AWS_SECRET_ACCESS_KEY")
 	awsAccessKeyID     = os.Getenv("AWS_ACCESS_KEY_ID")
 	awsVPCID           = "vpc-72464617"
-	workingDir         = "../.aws-terraform-tests"
 )
 
 var _ = BeforeSuite(func() {
-	err := os.MkdirAll(workingDir, os.ModePerm)
-	Expect(err).ToNot(HaveOccurred())
+	workingDir = GinkgoT().TempDir()
```

Comment:
Hi @gblue, I see that you have moved the terraform tests to work off a working dir. The reason that I hadn’t done it this way and chose to use a fixed path was just to avoid having to download providers amongst multiple runs.